src/services/SearchService.js

```diff
@@ -203,12 +365,16 @@ function handleSearchOrder(results, query){
 }
 
 // The skill search order, which has a secondary sort of the number of
-// Topcoder-verified skills, in descending order (where skillSource = ChallengeWin)
+// Topcoder-verified skills, in descending order (where level.name===verified)
 function skillSearchOrder(results, query){
   results = _.orderBy(results, [query.sortBy, function (member) {
-    challengeWinSkills = _.filter(member.emsiSkills, 
+    challengeWinSkills = _.filter(member.skills, 
       function(skill) {
-        return _.includes(skill.skillSources, 'ChallengeWin')
+        skill.levels.forEach(level => {
+          if(level.name==="verified"){
```

Comment:
For readability we should have spaces before and after the `===`

Reply:
Fixed